tools/devtool

```diff
@@ -438,6 +438,14 @@ cmd_build() {
     # Cargo uses the debug profile by default. If we're building the release
     # binaries, we need to pass an extra argument to cargo.
     cargo_args=("$@")
+
+    # When musl build are triggered, make sure that cargo finds the path to
+    # the musl-gcc binary.
+    TARGET_CC=""
+    if [[ ${cargo_args[@]} = *"musl"* ]]; then
+        TARGET_CC=/usr/bin/musl-gcc
```

Comment:
This path isn't fixed. I have it in `/usr/local/musl/bin//musl-gcc` for instance.

Reply:
Apparently there's no surefire portable way of finding a command's executable (easiest is the `which` command but `devtool` is supposed to run anywhere...including systems that don't have the `which` command) so you'd need to parse `$PATH` and look for it there.

Reply:
It's fine to have the path to `musl-gcc` hardcoded here - it's the path inside the container, not the one on the dev host. Apparently, the `cc` crate looks for the C compiler in the wrong place, so we need to force-point it to the right one.

Reply:
Aaah OK my bad. I didn't look inside the container.